tests/loader/loader-container-colors.html

```diff
@@ -22,7 +22,7 @@
             <div class="k-loader-container k-loader-container-lg k-loader-top">
                 <div class="k-loader-container-overlay k-overlay-dark"></div>
                 <div class="k-loader-container-inner k-loader-container-panel">
-                    <div class="k-loader k-loader-pulsing-2 k-loader-primary k-loader-lg">
+                    <div class="k-loader k-loader-pulsing-2 k-loader-lg k-loader-primary">
```

Comment:
you could reverse the order of the `size` and `themeColor` props in the component definition to avoid re-generating the tests